pages/docs/data-ops/cli.mdx

```diff
@@ -102,6 +102,84 @@ $ DATABASE_CONNECTION_NAME=dev glean preview
 
 Environment variable substitution is not yet supported for Builds that are triggered via a git revision.
 
+## Integration with dbt
+
+Glean can import models directly from dbt. This is useful for incorporating Glean into existing DataOps workflows, allowing you to make changes to your dbt schema while identifying issues in downstream visualizations and dashboards.
+
+You must add the following fields to the `meta` key of each dbt model you want imported to Glean:
+
+```yaml
+# schema.yml
+
+version: 2
+
+models:
+  - # ... dbt model configuration 
+    meta:
+      glean:
+        glean: "1.0" # Glean dbt integration version
+        name: # Glean model name
+        source:
+          connectionName: # which Glean data connection to use
```

Comment:
I would have the base case remove connection name from the spec.  An info callout / note should tell you to create a Glean database connection with the database name to have a good default workflow.